modules/oc-mirror-oci-format.adoc

```diff
@@ -12,6 +12,8 @@ You can use the oc-mirror plugin to mirror Operators in the Open Container Initi
 :FeatureName: Using the oc-mirror plugin to mirror Operator images in OCI format
 include::snippets/technology-preview.adoc[]
 
+When using this OCI feature, images are not automatically pruned from the target mirror registry.
```

Comment:
s/this/the 
Other references in this assembly are _the OCI feature_ or _the Technology Preview feature_. Using _this_ makes it sound like there are other OCI features. If so, feel free to ignore.